src/packages/version-check/src/version-check.ts

```diff
@@ -14,7 +14,7 @@ import type { VersionCheckOptions } from "./types";
 // Why is this not part of the config? Well, if a user
 // set this to a low value there could be problems.
 const ONE_DAY: number = 86400;
-
+const ACTIVATED = process.env.VC_ACTIVATED || false;
```

Comment:
I'd prefer if the behaviour around `process.env.VS_ACTIVATED` was more obvious (ie the type of ACTIVATED is `string | false`).

Maybe something like 
```suggestion
const isActivated = process.env.VC_ACTIVATED === "true";
```